sklearn/linear_model/randomized_l1.py

```diff
@@ -355,7 +355,8 @@ def _randomized_logistic(X, y, weights, mask, C=1., verbose=False,
 
     C = np.atleast_1d(np.asarray(C, dtype=np.float64))
     if C.ndim > 1:
-        raise ValueError("C should be 1-dim array, but got %d-dim array instead." % C.ndim)
+        raise ValueError("C should be 1-dim array, \
```

Comment:
If you decided to go for 1-dimensional use it everywhere (and that's a s and not a t).

Also you don't need the backslash:

```py
raise ValueError("beginning of the string "
    "that continues still here and with a format: {}".format("insert this")
```